build_html.sh

```diff
@@ -1,2 +1,3 @@
 chmod -R o+w source/
 docker run --rm -v $(pwd):/home/jovyan ubcdsci/py-intro-to-ds:20230104230634037f38 /bin/bash -c "jupyter-book build source"
+chmod -R 755 source/
```

Comment:
I generally go with the more semantic `o-w` to remove write perms. I'll fix that and merge